### PR TITLE
fix(build): remove cstyle check on cstor repo

### PR DESCRIPTION
**Pull request by kmova**
This repo pulls in the cstor repo as a dependency
and there is no need to run style checks on
the openebs/cstor repo.

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,7 +57,6 @@ install:
     - sh autogen.sh
     - ./configure --enable-debug --with-zfs-headers=$PWD/../cstor/include --with-spl-headers=$PWD/../cstor/lib/libspl/include
     - make -j4
-    - make -f ../cstor/Makefile cstyle CSTORDIR=$PWD/../cstor
```
Comment (vishnuitta) [inline .travis.yml:60]: `Makefile` in cstor directory gets created after running autogen.sh and configure commands in cstor directory.
So, this cstyle check can be moved down.
(or)
Below steps can be COPIED after the cstor branch is checked out few steps above
```
sh autogen.sh
./configure --with-config=user --enable-code-coverage=yes --enable-debug --enable-uzfs=yes --with-jemalloc --with-fio=$PWD/../fio --with-libcstor=$PWD/../libcstor/include
```
Above steps need to be done after checking out cstor repo

**Review by vishnuitta — APPROVED** [review]
changes are good

File: .travis.yml (hunk 1 above)
Comment (kmova) [reply .travis.yml:60]: oh. just did the same!

Comment (vishnuitta) [reply .travis.yml:60]: right.. :) exactly at the same time..
I added a comment, and I got a 'Refresh' link for PR

**Pull request merged**